File: app/src/main/java/net/treelzebub/zinepress/db/articles/ArticleCols.java
package net.treelzebub.zinepress.db.articles;

import android.provider.BaseColumns;

/**
 * Created by Tre Murillo on 2/19/16
 */
public interface ArticleCols extends BaseColumns {

    String _TABLE = "articles_table";

    String ID = "id";
    String DATE = "date";
    String TITLE = "title";
    String URL = "url";
}

File: app/src/main/java/net/treelzebub/zinepress/db/articles/ArticleCols.kt
package net.treelzebub.zinepress.db.articles

import android.provider.BaseColumns

/**
 * Created by Tre Murillo on 1/28/16
 */
object ArticleCols : BaseColumns {

    val _TABLE = "articles_table"

    val ID = "id"
    val DATE = "date"
    val TITLE = "title"
    val URL = "url"
}
